src/main/java/reposense/wizard/InputBuilder.java

```diff
@@ -0,0 +1,365 @@
+        input.append(ArgsParser.UNTIL_FLAGS[0] + WHITESPACE + date + WHITESPACE);
+        return this;
+    }
+
+    /**
+     * Adds the period flag with the {@code period} as argument to the input.
+     * This method should only be called once in one build.
+     *
+     * @param period The period.
+     */
+    public InputBuilder addPeriod(String period) {
+        input.append(ArgsParser.PERIOD_FLAGS[0] + WHITESPACE + period + WHITESPACE);
+        return this;
+    }
+
+    /**
+     * Adds the format flag with the {@code formats} as argument to the input.
+     * This method should only be called once in one build.
+     *
+     * @param formats The formats that need to be evaluated.
+     */
+    public InputBuilder addFormats(String formats) {
+        input.append(ArgsParser.FORMAT_FLAGS[0] + WHITESPACE + formats + WHITESPACE);
+        return this;
+    }
+
+    /**
+     * Adds the ignoreStandaloneConfig flag to the input.
+     * This method should only be called once in one build.
+     */
+    public InputBuilder addIgnoreStandaloneConfig() {
+        input.append(ArgsParser.IGNORE_CONFIG_FLAGS[0] + WHITESPACE);
+        return this;
+    }
+
+    /**
+     * Adds the ignoreFilesizeLimit flag to the input.
+     * This method should only be called once in one build.
+     */
+    public InputBuilder addIgnoreFilesizeLimit() {
+        input.append(ArgsParser.IGNORE_SIZELIMIT_FLAGS[0] + WHITESPACE);
+        return this;
+    }
+
+    /**
+     * Adds the timezone flag with the {@code zoneId} as argument to the input.
+     * This method should only be called once in one build.
+     */
+    public InputBuilder addTimezone(String zoneId) {
+        input.append(ArgsParser.TIMEZONE_FLAGS[0] + WHITESPACE + zoneId + WHITESPACE);
+        return this;
+    }
+
+    /**
+     * Adds the cloning threads flag with the {@code threads} as argument to the input.
+     * This method should only be called once in one build.
+     *
+     * @param threads The number of threads for cloning.
+     */
+    public InputBuilder addNumCloningThreads(int threads) {
+        input.append(ArgsParser.CLONING_THREADS_FLAG[0] + WHITESPACE + threads + WHITESPACE);
+        return this;
+    }
+
+    /**
+     * Adds the analysis threads flag with the {@code threads} as argument to the input.
+     * This method should only be called once in one build.
+     *
+     * @param threads The number of threads for analysis.
+     */
+    public InputBuilder addNumAnalysisThreads(int threads) {
+        input.append(ArgsParser.ANALYSIS_THREADS_FLAG[0] + WHITESPACE + threads + WHITESPACE);
+        return this;
+    }
+
+    /**
+     * Adds the flag to enable shallow cloning.
+     * This method should only be called once in one build.
+     */
+    public InputBuilder addShallowCloning() {
+        input.append(ArgsParser.SHALLOW_CLONING_FLAGS[0] + WHITESPACE);
+        shallowCloning = true;
+        return this;
+    }
```

Comment:
For future PRs, we might want to consider cleaning up any unused inputbuilder methods.